src/models/launch_lite_1/README.md

```diff
@@ -41,4 +41,4 @@ The System76 Launch Lite is a configurable keyboard with the following specifica
   - 1x USB Type-C to USB Type-C
   - 1x USB Type-C to USB Type-A
 - Dimensions
-  - 30.9cm x 13.6cm x 3.3cm, 948g
+  - 30.65cm x 10.75cm x 3.06cm, 816.46g
```

Comment:
> 30.65cm x 10.75cm x 3.06cm, 816.46g

The website's tech specs and product card list the length as 30.83 cm (308.3 mm), which makes sense because it matches the regular Launch's width. Looks like its width and height don't match these numbers yet, either.

Reply:
Please push a commit to fix it however you think is best.

Reply:
The schematic shows the 30.83 cm that I'd expect (looks like the website matches the schematic now), so I'll go with what's on the schematic and website.

Reply:
Fixed with https://github.com/system76/tech-docs/pull/157/commits/af0a82f4ba0a2f2bc1e24423d0025b09e8c1f900.